app/adapters/application.js

```diff
@@ -20,7 +19,7 @@ export const fixFilterQuery = query  => {
   return query;
 };
 
-export default JSONAPIAdapter.extend(HasManyQueryAdapterMixin, AdapterFetch, FastbootAdapter, {
+export default JSONAPIAdapter.extend(HasManyQueryAdapterMixin, FastbootAdapter, {
```

Comment:
Unrelated changes are appearing in your PR, Please rebase correctly.

Reply:
@kushthedude, can you please guide me on gitter. I'm having some problems related to rebase. That would be so helpful.